### DISCUSSION TITLE
Help to migrate from old version to newversion

### DISCUSSION THREAD

**Discussion by Irfadg** [flutter-stripe/flutter_stripe#1500] [Q&A]
Hi,

currently I'm using  flutter_stripe:4.0.0. now I wanted to update in to  latest version (9.5.0).

then how to solve these issue .

-The method 'presentApplePay' isn't defined for the type 'Stripe'.
Try correcting the name to the name of an existing method, or defining a method named 'presentApplePay'.

-The method 'confirmApplePayPayment' isn't defined for the type 'Stripe'.
Try correcting the name to the name of an existing method, or defining a method named 'confirmApplePayPayment'.

-The getter 'isApplePaySupported' isn't defined for the type 'Stripe'.
Try importing the library that defines 'isApplePaySupported', correcting the name to the name of an existing getter, or defining a getter or field named 'isApplePaySupported'.

how can i replace  these presentApplePay, confirmApplePayPayment, isApplePaySupported.

for old version these are supporting , in newer version these code's are deprecated .

9.4.0 [#](https://pub.dev/packages/flutter_stripe/changelog#940)
Breaking changes-

Removed deprecated methods isApplePaySupported, checkApplePaySupport, presentApplePay, confirmApplePayPayment.

thankyou

**Comment by remonh87** [comment]
Check this document: https://docs.page/flutter-stripe/flutter_stripe/platform_pay_migration
